pom.xml

```diff
@@ -309,6 +309,17 @@
     </extraJavaTestArgs>
   </properties>
   <repositories>
+    <repository>
```

Comment:
We'd remove this before merging? after it's released

Reply:
Yes, will remove this section once the official 3.3.2 release is out.